mmv1/third_party/terraform/tests/resource_compute_backend_service_test.go.erb

```diff
@@ -731,6 +731,60 @@ func TestAccComputeBackendService_withCompressionMode(t *testing.T) {
 	})
 }
 
+func TestAccComputeBackendService_trafficDirectorLbPolicies(t *testing.T) {
```

Comment:
I think we can remove this test, since both `testAccComputeBackendService_trafficDirectorLbPolicies` and `TestAccComputeBackendService_trafficDirectorUpdateLbPolicies` call the same config for resource creation.

Beside from this LGTM.

Reply:
Agreed, I removed that test.

What is the next step, will you be merging the PR?

Reply:
Thanks! 
Once the tests pass I will approve the PR. Riley will do an additional round of review to double-check. Then we can merge the PR.